Yoast/YoastSEO.js
# Add ARIA attributes to the SVG icons.

## Summary

This PR can be summarized in the following changelog entry:

* Improve SVG images accessibility

## Relevant technical choices:

* Improve SVG images accessibility

## Test instructions

As done in https://github.com/Yoast/wordpress-seo/pull/9276, https://github.com/Yoast/yoast-components/pull/280 and https://github.com/Yoast/my-yoast/pull/1549

- adds attributes role="img" aria-hidden="true" focusable="false" to the SVG images
- changes are applied to all the SVGs I was able to find, even the ones used as backgrounds so they're all standardized to the same format and to make sure there are no SVGs with pointless tags/attributes around

To test:
- check the SVG icons in the browserified example still look OK

Fixes #1240
